Build-OpenCV.sh

```diff
@@ -221,9 +213,6 @@ if [[ -n $(echo $PATH | grep 'conda') ]] ; then
 
         rename_so libfontconfig.so
         rename_so libpangoft2-1.0.so
-        rename_so libz.so
-        rename_so libfreetype.so
-        rename_so libharfbuzz.so
```

Comment:
Each rename was added after build failed on 16.04 in Travis, so all of them will mostly be needed
Adding an OS version check for the ones specific for 16.04 would be good though

Reply:
The others are warnings, so might be okay to remove these